src/cli.ts

```diff
@@ -160,6 +160,13 @@ export const cliOptions = {
     default: true,
     describe: 'Set to false to exclude tools related to network.',
   },
+  screenshotFormat: {
+    type: 'string',
+    describe:
+      'Default image format for screenshots. Options: png, jpeg, webp. Default is png.',
```

Comment:
Let's clarify that this is for screenshots taken by take_screenshot if no format is supplied to the tool call.

Reply:
done